docs/MSPX.md

```diff
@@ -0,0 +1,18 @@
+# mLRS Documentation: MspX #
+
+([back to main page](../README.md))
+
+**Important: MSP support is experimental and currently only available in the branch 'dev-mspx'.**
+
+MspX is a technology designed to improve the over-the-air communication for systems using the MSP protocol, such as INAV systems. It introduces more robust framing and parsing which reduces packet losses to the minimum, and adds features such as compression of some very large MSP messages. It also converts MSP messages into CRSF telemetry messages to the radio, thus providng telemetry to the radio and Lua scripts running on the radio, such as the INAV app.
+
+MspX is currently under test with INAV 7.1.
```

Comment:
Should this be up on the page further (or part of the Important Disclaimer)?

Reply:
changed the sentence, wasn't maybe what I meant
:)